package/MDAnalysis/coordinates/TRJ.py

```diff
@@ -458,74 +463,164 @@ class NCDFReader(base.ReaderBase):
              'length': 'Angstrom',
              'velocity': 'Angstrom/ps',
              'force': 'kcal/(mol*Angstrom)'}
+
     _Timestep = Timestep
 
     def __init__(self, filename, n_atoms=None, mmap=None, **kwargs):
+
+        def verify_units(eval_unit, expected_units):
```

Comment:
Can you make this function a static method `_verify_units()` instead of hiding it?

You can then also explicitly test it.

Reply:
This should be handled in 54da919, let me know if you were thinking of something else/additional tests though.